modules/tests/shared/src/test/scala/lucuma/core/math/UnitsSuite.scala

```diff
@@ -17,33 +17,33 @@ final class UnitsSuite extends FunSuite {
 
   test("Sanity tests") {
     assert(Rational.one.withUnit[Angstrom] === Rational(1, SafeLong(10).pow(10)).withUnit[Meter])
-    // assert(
-    //   Rational.one.withUnit[Jansky] === Rational(1, SafeLong(10).pow(26))
-    //     .withUnit[Watt %/ (Meter %^ 2) %* Hertz]
-    // )
-    // assert(Rational.one.withUnit[Erg] === Rational(1, SafeLong(10).pow(7)).withUnit[Joule])
+    assert(
+      Rational.one.withUnit[Jansky] === Rational(1, SafeLong(10).pow(26))
+        .withUnit[Watt / (Meter ^ 2) * Hertz]
+    )
+    assert(Rational.one.withUnit[Erg] === Rational(1, SafeLong(10).pow(7)).withUnit[Joule])
   }
 
-  // test("Check brightness unit names") {
-  //   assertEquals(1.withUnit[VegaMagnitude].show, "1 Vega mag")
-  //   assertEquals(1.withUnit[ABMagnitude].show, "1 AB mag")
-  //   assertEquals(1.withUnit[Jansky].show, "1 Jy")
-  //   assertEquals(1.withUnit[WattsPerMeter2Micrometer].show, "1 W/m²/µm")
-  //   assertEquals(1.withUnit[ErgsPerSecondCentimeter2Angstrom].show, "1 erg/s/cm²/Å")
-  //   assertEquals(1.withUnit[ErgsPerSecondCentimeter2Hertz].show, "1 erg/s/cm²/Hz")
-  //   assertEquals(1.withUnit[VegaMagnitudePerArcsec2].show, "1 Vega mag/arcsec²")
-  //   assertEquals(1.withUnit[ABMagnitudePerArcsec2].show, "1 AB mag/arcsec²")
-  //   assertEquals(1.withUnit[JanskyPerArcsec2].show, "1 Jy/arcsec²")
-  //   assertEquals(1.withUnit[WattsPerMeter2MicrometerArcsec2].show, "1 W/m²/µm/arcsec²")
-  //   assertEquals(
-  //     1.withUnit[ErgsPerSecondCentimeter2AngstromArcsec2].show,
-  //     "1 erg/s/cm²/Å/arcsec²"
-  //   )
-  //   assertEquals(1.withUnit[ErgsPerSecondCentimeter2HertzArcsec2].show, "1 erg/s/cm²/Hz/arcsec²")
-  //   assertEquals(1.withUnit[WattsPerMeter2].show, "1 W/m²")
-  //   assertEquals(1.withUnit[ErgsPerSecondCentimeter2].show, "1 erg/s/cm²")
-  //   assertEquals(1.withUnit[WattsPerMeter2Arcsec2].show, "1 W/m²/arcsec²")
-  //   assertEquals(1.withUnit[ErgsPerSecondCentimeter2Arcsec2].show, "1 erg/s/cm²/arcsec²")
-  // }
+  test("Check brightness unit names") {
+    assertEquals(1.withUnit[VegaMagnitude].show, "1 Vega mag")
+    assertEquals(1.withUnit[ABMagnitude].show, "1 AB mag")
+    assertEquals(1.withUnit[Jansky].show, "1 Jy")
+    assertEquals(1.withUnit[WattsPerMeter2Micrometer].show, "1 W/m²/µm")
```

Comment:
@erikerlandson ok sorry, next dumb question. Any thoughts on this?
```
=> Diff (- obtained, + expected)
-1 W/(m^2 ((Micro * Meter)))
+1 W/m²/µm
```
@cquiroz how particular are you about the formatting of these?

Reply:
We use them in the UI in some places

Reply:
But we could find perhaps other ways to do it

Reply:
that result looks kind of weird.  In general it is probably dangerous to assume too much about how `show` renders but I'd expect it to not be printing `Micro` and `Meter` out there.

Reply:
is micrometer also a type alias?

Reply:
The abbreviation is made explicit when defining the units. See https://github.com/gemini-hlsw/lucuma-core/blob/master/modules/core/shared/src/main/scala/lucuma/core/math/units.scala#L78

Maybe `WattsPerMeter2Micrometer` can be an opaque type in Scala 3?

Reply:
aha, `UnitStringDef` is no longer a thing in scala3 coulomb, however when defined using the new variation it should still be explicit.  And yes, `Micrometer` is a type alias.  I see, `WattsPerMeter2Micrometer` is not a `DerivedUnit`.

I think I know what's going on, it's something similar to canonical sig, but over in the `show` metaprogramming.

Reply:
> however when defined using the new variation it should still be explicit

@erikerlandson sorry, wdym by this? :)

Reply:
@armanbilge  I was assuming `WattsPerMeter2Micrometer` was a `DerivedUnit`, where you can tell it what to use for an abbv.  But it is a `type = ` alias.

Reply:
I see, so that would be the way to define it?

Reply:
I think I can provide decent support for type aliases.  If you want *total* control over how you define the abbreviations, then declaring it as a `DerivedUnit` is probably the best practice.

Reply:
Thanks, that's really helpful :) @cquiroz I guess a `DerivedUnit` is probably the way to go here.

Reply:
@erikerlandson I'm kind of worried `UnitStringDef` doesn't exist (at least implicitly) in Scala 3. We rely on it to have a runtime representation of units. (see https://github.com/gemini-hlsw/lucuma-core/blob/master/modules/core/shared/src/main/scala/lucuma/core/math/dimensional/unit.scala).

Actually, it would help us a lot if coulomb supported runtime representation of units out of the box. We have cases in which users can specify units for a quantity and we want to store the quantity in the specified units + the actual units.

Reply:
@rpiaggio  I can support it, if people want one.   I was excited about being able to instantiate the strings directly in-line, without needing the `UnitString` as an intermediary, but any method that creates things in-line can also populate a corresponding implicit.

Reply:
Our runtime `Units` representation leverages the fact that `UnitString` is a "de facto" runtime representation. We don't need `UnitString` per se, we wouldn't need it if coulomb supported runtime units by design. Do you think that's a useful feature for coulomb?

Reply:
@rpiaggio it is certainly possible to do.  My entire philosophy around coulomb is unit analysis as a compile-time type checking system, so I haven't given a lot of thought to it.  However it is in many ways easier to do it at run-time.

I think the tricky part would be to somehow leverage static unit definitions in a run-time system.  The only alternative would be two totally separate unit definition systems, which seems inelegant and wasteful.

Reply:
An example of runtime units which work mostly like coulomb units do, but at runtime, is the python 'pint' package:
https://pint.readthedocs.io/en/0.6/

Reply:
> I think the tricky part would be to somehow leverage static unit definitions in a run-time system.  The only alternative would be two totally separate unit definition systems, which seems inelegant and wasteful.

@erikerlandson relevant reading I think: https://hirrolot.github.io/posts/why-static-languages-suffer-from-complexity

Reply:
@armanbilge  that is a nice articulation of the problem, and gives me a sad  :joy: 

So in practical terms,
1. I can re-implement `UnitString`, and allow whatever your current scheme is to work as it was
2. A parallel runtime (pint-like?) system seems possible, but is obviously a longer-term project. 
3. Deriving a pint-style 'unit registry' from static unit types seems fruitful.

Reply:
`UnitString` nicely materializes runtime representations of units defined in the type-level, or at least a starting point.

If you wish, you can integrate our `Units` and `Measure` classes into a module in coulomb, which would give it the ability to handle runtime units. (`Measure` is a version of `Quantity` with runtime units).

Reply:
Latest `scala3` branch adds a `ShowUnit` such that `summon[ShowUnit[U]].value` yields the unit string.
You can also define the unit strings for type aliases using `ShowUnitAlias`:

https://github.com/erikerlandson/coulomb/blob/scala3/core/src/test/scala/coulomb/quantity.scala#L558

Reply:
@erikerlandson nice, thank you! So is that the new "blessed" way, instead of using derived units like I did in 963ee5927707d65bab2ecdf4b2edfa755531a7b9?

Reply:
The mental image I had in mind when designing coulomb was that units having a standard name, like `Angstrom` or `Jansky` or `Erg`, would be defined using `DerivedUnit`

Something like  `ErgsPerSecondCentimeter2Angstrom`  would just be unit expressions: `Erg / (Second * ((Centi * Meter) ^ 2) * Angstrom)` - I don't (personally) understand the point of defining a type name for that.

I guess I consider `DerivedUnit` to be "more blessed" than `type X = `, but ultimately it's a matter of preferred style.  I don't mind supporting `type X = ...`